src/components/Search/Search.tsx

```diff
@@ -1,42 +1,38 @@
 import React, { useState } from 'react';
 import './Search.css';
 
-const Search = () => {
+interface getMarkets {
```

Comment:
I didn't realize we need to declare an interface for props!  Also - good to know that we can make an interface that is a function.